packages/nvidia_nat_adk/src/nat/plugins/adk/llm.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nat.builder.builder import Builder
+from nat.builder.framework_enum import LLMFrameworkEnum
+from nat.cli.register_workflow import register_llm_client
+from nat.llm.litellm_llm import LiteLlmModelConfig
+
+
+@register_llm_client(config_type=LiteLlmModelConfig, wrapper_type=LLMFrameworkEnum.ADK)
+async def litellm_adk(litellm_config: LiteLlmModelConfig, builder: Builder):
+
+    from google.adk.models.lite_llm import LiteLlm
+
+    config_obj = {
+        **litellm_config.model_dump(exclude={"type"}, by_alias=True),
+    }
+
+    llm = LiteLlm(
+        model=config_obj.get("model"),
```

Comment:
I am seeing a linter error here. Maybe a check like this is needed - 
   if not config_obj.get("model"):
       raise ValueError("model_name is required for LiteLLM ADK")